Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/main/home/HomeFragmentViewModel.kt

```diff
@@ -221,14 +224,22 @@ class HomeFragmentViewModel @AssistedInject constructor(
     ) { tracingItem, submissionItem, submissionState, statsData ->
         mutableListOf<HomeItem>().apply {
             when (submissionState) {
-                TestPositive, SubmissionDone -> {
+                TestPositive, is SubmissionDone -> {
                     // Don't show risk card
                 }
                 else -> add(tracingItem)
             }
 
             add(submissionItem)
 
+            if (submissionState is SubmissionDone) {
```

Comment:
Shouldn't we show the card once the user has seen the positive test result? The ticket says: Once the user got a positive test result, the risk calculation shall be stopped.
Submission done comes later and may never be reached.

Reply:
No, the ticket says that we should inform users after they share their keys

Reply:
Yes, you are right